### ISSUE TITLE
Has anyone been able to get this to work?

### ISSUE THREAD

**Issue by samr28** [bhberson/InspiroBotSlack#1]
I can get app.js to run fine with node and everything looks normal (localhost:3000 shows Hello World!). I have added the path env var. When I send messages to the slack channel, the bot doesn't seem to pick them up. Maybe I did something wrong?

**Comment by apalanki** [comment]
Yes, it works. For it to work, your app needs to be deployed somewhere public and Slack's webhook needs to be configured to use the public URL as pointed out in README.
